.github/workflows/test.yml

```
@@ -69,6 +71,115 @@ jobs:
         spec: cypress/integration/basic/**/*
         suffix: "basic"
 
+  jekyll_bundler_run: 
+    runs-on: ubuntu-latest
+    outputs:
+      sha: ${{ steps.run.outputs.sha }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/cache@v2
+      with:
+        path: sample_site_gemfiles/vendor/bundle
+        key: gems-${{ hashFiles('**/Gemfile') }}
+        restore-keys: |
+          gems
+    - name: Jekyll bundler run
+      id: run
+      uses: ./
+      with:
+        jekyll_build_options: "--config ../sample_site/_config.yml,../sample_site/_config_jekyll_bundler.yml"
+        jekyll_src: sample_site
+        target_branch: jekyll_bundler_test_pages_branch
+        jekyll_env: production
+        bundler_version: 2.3.11
+        github-token: ${{secrets.JEKYLL_PAT}} 
```

My bad, sorry I got confused with the `github-script` action.
```suggestion
        token: ${{secrets.JEKYLL_PAT}} 
```